node_modules/heroku-cli-util/test/styled.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+let expect = require('chai').expect;
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'require' is not defined.